README.md

```diff
@@ -11,26 +11,54 @@ This project is a moblie website for personal portfolio. Home page of this proje
 Major languages:- HTML, CSS
 Technologies:- VScode
 
-## Getting Started
+## Project Link:- https://jusha25.github.io/personal-portfolio/
+
+## Getting Started:-
+
+To get a local copy up and running, follow the steps in Setup below.
+
+## Setup
+
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+[Download the ZIP from this location](https://github.com/Jusha25/personal-portfolio/archive/refs/heads/master.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
+
+```bash
 
-To view this project, download the repo from github and run on the local machine browser.
+git clone https://github.com/Jusha25/personal-portfolio.git
+
+- Once the files are on your machine, open the _personal-portfolio_ folder in [Visual Studio Code](https://code.visualstudio.com/)
+
+
+- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).
+
+## Visit And Open Files
+https://github.com/Jusha25/personal-portfolio
 
 ## Authors
 
 👤 Author1
 
+Shaili Sharma
+
 GitHub: https://github.com/Jusha25
-Twitter: @shailisharma
-LinkedIn: shailisharma
+Twitter: @shailisharma25
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT && RECOMENDED** Good job making all the changes required by the previous review. But I noticed that your README file is not looking in a professional way. I suggest you use this [REAME](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) as a template(**Click [here](https://github.blog/changelog/2021-09-20-quickly-copy-the-contents-of-a-file-to-the-clipboard/#:~:text=To%20copy%20the%20contents%20of,more%20about%20working%20with%20files.)** and see how to copy everything in a Github file). Once you have everything paste it into your README file and edit according your project's info.

**NOTICE**: Make sure you did this change before maerging your pull request

Reply:
Yes, I have seen that, and thanks a lot for approving my project.
Happy Coding,